modules/smart-agent_kubernetes-workloads-count/conf/readme.yaml

```diff
@@ -0,0 +1,22 @@
+documentations:
+  - name: Kubernetes Deployment
+    url: 'https://kubernetes.io/docs/concepts/workloads/controllers/deployment/'
+source_doc: >-
+  We are using metrics from the
```

Comment:
I think you can remove this sentence (or link to the "kubernetes-common" readme which should contain everything needed).

in addition to linking the right monitor in related doc (see comment above) this should be enough